### PR TITLE
[bugfix]: fix DistEvalHook missing vis_config

### PR DESCRIPTION
**Pull request by Cathy0908**

### CODE REVIEW

File: easycv/hooks/eval_hook.py [hunk 1]
```diff
@@ -148,6 +148,9 @@ def __init__(self,
         self.eval_kwargs = eval_kwargs
         self.initial = initial
         self.flush_buffer = flush_buffer
+        # hook.evaluate runs every interval epoch or iter, popped at init
+        self.vis_config = self.eval_kwargs.pop('visualization_config', {})
+        self.gpu_collect = self.eval_kwargs.pop('gpu_collect', None)
```
Comment (haiasd) [inline easycv/hooks/eval_hook.py:153]: 加上这个会导致gpu_collect一直是false

Comment (wenmengzhou) [reply easycv/hooks/eval_hook.py:153]: self.gpu_collect = self.eval_kwargs.pop('gpu_collect', gpu_collect)

Comment (wenmengzhou) [reply easycv/hooks/eval_hook.py:153]: delete line 51 and 146 
self.gpu_collect = self.eval_kwargs.pop('gpu_collect', gpu_collect)

Comment (Cathy0908) [reply easycv/hooks/eval_hook.py:153]: EvalHook has no param of gpu_collect,cannot delete 51

Comment (Cathy0908) [reply easycv/hooks/eval_hook.py]: done!

**Review by wenmengzhou — APPROVED** [review]

**Pull request merged**